lib/generators/action_view/component/templates/component.rb.tt

```diff
@@ -0,0 +1,7 @@
+class <%= class_name %> < ActionView::Component::Base
+  validates :content, presence: true
```

Comment:
Should we make this configurable too? Because I don't think all components have `content`

Reply:
I'm not sure what the best interface for communicating whether the component has content or not would be. We could use thor to ask for user input indicating if they want their component to have it or not. Something like:

```ruby
def should_add_content?
  ask("Would you like your component to have children? (Y/n)").downcase == "y"
end
```

Reply:
If we can do a check like ☝️ easily, I think it would be a nice touch ❤️ 

Suggested copy: `Would your like MyComponent to require content?`